src/main/scala/org/broadinstitute/dsde/firecloud/model/Profile.scala

```diff
@@ -10,6 +10,11 @@ case class ThurloeKeyValue(
   userId: Option[String] = None,
   keyValuePair: Option[FireCloudKeyValue] = None)
 
+case class ThurloeNotification(
+  userId: String,
+  notificationId: String,
+  substitutions: Map[String, String])
+
```

Comment:
@MatthewBemis we had talked through a more strongly typed way of modeling notifications, what happened to that?